config/locales/en.yml

```diff
@@ -74,3 +74,17 @@ en:
       signup: "Sign up"
       content: "This will be a signup page for new users."
       signup2: "Sign up now!"
+      name: "Name"
+      pass: "Password"
+      confirm: "Confirmation"
+      btn: "Create my account"
+    create:
+      welcome: "Welcome to the Sample App!"
+    show:
+      danger: "User not exists!"
+  shared:
+    error_messages:
+      content: "The form contain"
```

Comment:
nên gộp lại và khai báo 1 câu thôi nha em, vì vị trí có thể đảo tùy quốc gia nên ko phân tách trên cùng 1 câu
count: 
 one: "The form contain %{count} error"
 other: "The form contain %{count} errors"

Reply:
ok chi